src/clients/ProxyProtocolWriter.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 1996-2025 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_SRC_CLIENTS_PROXYPROTOCOLWRITER_H
+#define SQUID_SRC_CLIENTS_PROXYPROTOCOLWRITER_H
+
+#include "base/AsyncCallbacks.h"
+#include "base/AsyncJob.h"
+#include "base/CbcPointer.h"
+#include "clients/forward.h"
+#include "CommCalls.h"
+#include "comm/Connection.h"
+#include "http/StatusCode.h"
+#include "sbuf/SBuf.h"
+
+class ErrorState;
+class AccessLogEntry;
+typedef RefCount<AccessLogEntry> AccessLogEntryPointer;
+
+/// Proxy protocol header sending results (supplied via a callback).
+class ProxyProtocolWriterAnswer
```

Comment:
I could not find a reason why we should move this class into a separate module. A similar TunnelerAnswer class was  added separately from Http::Tunneler. (initially at f5e179). I looked through PR380 comments but could not find any discussion/concerns regarding this separation.

Reply:
Yes, it looks like it was born that way, possibly to avoid future change requests.